code/analytics/src/main/java/com/adobe/marketing/mobile/analytics/internal/AnalyticsHitProcessor.kt

```diff
@@ -156,7 +158,7 @@ internal class AnalyticsHitProcessor(
 
                     // Dispatch response only if the hit was sent after the reset Identities was called.
                     // So that we only populate UUID if the hit was sent after reset in case where AAMForwarding is enabled.
-                    if (timestamp > analyticsState.lastResetIdentitiesTimestamp) {
+                    if (timestamp > analyticsState.lastResetIdentitiesTimestampSec) {
```

Comment:
👍  we should also update param names and document the timestamp is in seconds in AnalyticsHit and AnalyticsHitDatabase#queue